test/test_plot_collection.py

```diff
@@ -70,9 +70,21 @@ def test_remove_nonexistent_plot(self):
             plot_coll.remove("plot_3")
         except KeyError:
             pass
-        assert len(plot_coll.plotters) == 1
```

Comment:
What is this testing for with the assertion removed?

Reply:
nice catch, that was an accidental removal